C7FIT/eBayItem.swift

```diff
@@ -0,0 +1,52 @@
+//
+//  eBayItem.swift
+//  C7FIT
+//
+//  Created by Brandon Lee on 3/5/17.
+//  Copyright © 2017 Brandon Lee. All rights reserved.
+//
+
+import Foundation
+
+/**
+    A model representation of an eBay item
+ */
+struct eBayItem {
+    let mainImage: String?
+    var additionalImages = [String?]()
```

Comment:
are these URLs? if so, either a better name for the var or maybe convert them to URL type